client/cypress/fixtures/data.json

```diff
@@ -0,0 +1,10 @@
+{
+  "username": "jenaye",
+  "password": "jenaye",
+  "baseUrl": "http://localhost:4200",
+  "backendUrl": "http://localhost:8000",
+  "dashboardRoute": "/dashboard",
+  "loginRoute": "/login",
+  "missionsRoute": "/missions",
+  "vulnerabilitiesRoute": "/vulnerabilities"
+}
```

Comment:
Add new line at the end of the file

Reply:
@Darkweak  thank's , it will be merged or i  still have to make this tiny modification before ?

Reply:
We won't merge until the last fix is done because we don't want to increase the reminder/todo list :)

Reply:
@Darkweak Done !